src/ripple/app/main/LoadManager.cpp

```diff
@@ -72,22 +85,24 @@ class LoadManagerImp
     //
     //--------------------------------------------------------------------------
 
-    void onPrepare ()
+    void onPrepare () override
     {
     }
 
-    void onStart ()
+    void onStart () override
     {
-        m_journal.debug << "Starting";
-        startThread ();
+        JLOG(mJournal.debug) << "Starting";
+        assert (! mThread.joinable());
+
+        mThread = std::thread {&LoadManagerImp::run, this};
```

Comment:
Howard's rationale is compelling, and now that we're on VS2015 there are fewer internal compiler errors with brace initialization. I hope.